packages/web-components/src/components/footer/footer.scss

```diff
@@ -254,7 +259,14 @@
 
   @include carbon--breakpoint('lg') {
     order: 1;
+    flex: 0 0 25%;
+    max-width: 20rem;
+    min-width: 13.5rem;
```

Comment:
Thanks @NSDrowned! We can avoid hard-coding the width values by using `carbon--mini-units()`.

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/0038432f7fa8b2596b403a534b2f42c527b3f122/packages/styles/scss/components/footer/_language-selector.scss#L51-L52

Reply:
Nice! I didn't know that :)

Reply:
@NSDrowned I see you resolved this thread but I don't see the change, perhaps forgot to push?

Reply:
@jeffchew forgot to push 🤦